safeeyes/safeeyes.py

```diff
@@ -122,10 +130,8 @@ def start(self):
             self.safe_eyes_core.start()
             self.handle_system_suspend()
 
-        self.run()
-
-    def do_startup(self):
-        Gtk.Application.do_startup(self)
+    def do_activate(self):
+        logging.info('Application activated')
```

Comment:
When I run safeeyes from terminal, I don't get any terminal message like "Application activated". Is that expected behavior?

Reply:
Did you run with `--debug`? Info messages don't show by default.

Reply:
works!